components/OgImage/EventShareInstagram.vue

```
@@ -1,148 +1,95 @@
-  "SET",
-  "OUT",
-  "NOV",
-  "DEZ",
-] as const;
+const monthNames = [
+  'janeiro','fevereiro','março','abril','maio','junho',
+  'julho','agosto','setembro','outubro','novembro','dezembro'
+] as const
 
-const validDate = props.startAt ? new Date(props.startAt) : undefined;
+const validDate = props.startAt ? new Date(props.startAt) : undefined
```

ESLint

[error] Insert `;`

(prettier/prettier)